torch/testing/_internal/opinfo/definitions/_masked.py

```diff
@@ -566,6 +562,9 @@ def sample_inputs_masked_normalize(op_info, device, dtype, requires_grad, **kwar
             DecorateInfo(
                 unittest.skip("Skipped!"), "TestJit", "test_variant_consistency_jit"
             ),
+            # RuntimeError: "prod_cpu" not implemented for 'BFloat16'
+            DecorateInfo(unittest.expectedFailure, 'TestDecomp', 'test_comprehensive',
+                        dtypes=(torch.bfloat16,), device_type='cpu'),
```

Comment:
Regression: The new path (with Python Dispatch) uses `prod` which doest not support BFloat16 on CPU.